content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -288,23 +288,28 @@ Always、OnFailure 和 Never。默认值是 Always。
 
 A Pod has a PodStatus, which has an array of
 [PodConditions](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podcondition-v1-core)
-through which the Pod has or has not passed:
+through which the Pod has or has not passed. Kubelet manages the following
+PodConditions:
 -->
 ## Pod 状况  {#pod-conditions}
 
 Pod 有一个 PodStatus 对象，其中包含一个
 [PodConditions](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podcondition-v1-core)
 数组。Pod 可能通过也可能未通过其中的一些状况测试。
+Kubelet 管理以下 PodCondition：
 
 <!--
 * `PodScheduled`: the Pod has been scheduled to a node.
+* `PodHasNetwork`: (alpha feature; must be [enabled explicitly](#pod-has-network)) the
+  Pod sandbox has been successfully created and networking configured.
 * `ContainersReady`: all containers in the Pod are ready.
 * `Initialized`: all [init containers](/docs/concepts/workloads/pods/init-containers/)
   have completed successfully.
 * `Ready`: the Pod is able to serve requests and should be added to the load
   balancing pools of all matching Services.
 -->
 * `PodScheduled`：Pod 已经被调度到某节点；
+* `PodHasNetwork`：Pod 沙箱被成功被创建并且配置了网络（Alpha 功能，必须[显式启用] (#pod-has-network)）；
```

Comment:
The `[text](link)` syntax can have space between, but I think it's not recommended.

```suggestion
* `PodHasNetwork`：Pod 沙箱被成功被创建并且配置了网络（Alpha 功能，必须[显式启用](#pod-has-network)）；
```